### ISSUE TITLE
Most articles in Storefront blog break width on mobile

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#2786]
Works on mobile here: https://www.designsystemet.no/bloggen/2024/v1rc2

Breaks width and unable to scroll x: https://www.designsystemet.no/bloggen/2024/v1rc1
This is also the case for older blog articles. ^
![Image](https://github.com/user-attachments/assets/2eb5a35e-c410-4103-b01a-69b8d9c43634)
